lib/tasks/local_dev.rake

```diff
@@ -1,15 +1,30 @@
 desc 'Set up your local development environment with data from Find'
 task setup_local_dev_data: %i[environment copy_feature_flags_from_production sync_dev_providers_and_open_courses] do
   puts 'Creating a provider-only user with DfE Sign-in UID `dev-provider` and email `provider@example.com`...'
-  ProviderUser.find_or_create_by!(dfe_sign_in_uid: 'dev-provider', email_address: 'provider@example.com') do |u|
+  ProviderUser.find_or_create_by!(
+    dfe_sign_in_uid: 'dev-provider',
+    email_address: 'provider@example.com',
+    first_name: 'Apply',
+    last_name: 'Provider',
+  ) do |u|
     u.providers = Provider.where(code: '1JA').all
   end
 
   puts 'Creating a support & provider user with DfE Sign-in UID `dev-support` and email `support@example.com`...'
 
-  SupportUser.find_or_create_by!(dfe_sign_in_uid: 'dev-support', email_address: 'support@example.com')
+  SupportUser.find_or_create_by!(
+    dfe_sign_in_uid: 'dev-support',
+    email_address: 'support@example.com',
+    first_name: 'Apply',
+    last_name: 'Support',
```

Comment:
Can these users ever exist already? In that case this would crash because it'll try to create a new record with an existing email/dsi-uid.

Reply:
I don't 🤔 not sure why you'd run this without a fresh db

Reply:
Fair. Can we change it to `create!` then?